source_gen/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@
   not be `.g.dart`. To produce `.g.dart` files please use the
   `SharedPartBuilder`.
 
+* `GeneratorForAnnotation.generateForAnnotatedElement` now allow multiple return
+  values when implementations return an `Iterable` or `Stream`.
+  * Values from `generateForAnnotatedElement` have whitespace trimmed. `null`
```

Comment:
its it worth allowing nulls? that seems like its just hiding errors

Reply:
oh so maybe allow a top level null but not individual `null` values in the Stream/Iterable then?

Reply:
Hrm...

Well, I kinda like `null`

I could have this in my generator:

```dart
yield _myHelper(context);
```

Then `_myHelper` is designed to return null on no-op.

Reply:
sure I don't think that really promotes good patterns, but :man_shrugging: